modules/ToDoLogic.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable import/prefer-default-export */
+import '../src/style.css';
+import dots from '../src/images/dots-icon.png';
+import reset from '../src/images/reset-icon.png';
+
+const $input = document.querySelector('.input');
+const $listTasks = document.querySelector('.list-tasks');
+const $titleContainer = document.querySelector('.title-container');
+
+const img = document.createElement('img');
+$titleContainer.appendChild(img);
+img.src = reset;
+img.classList.add('reset');
+const $reset = document.querySelector('.reset');
+
+class Task {
+  constructor(description) {
+    this.description = description;
+    this.completed = false;
+    this.index = 0;
```

Comment:
- [ ] This project milestone requires you to create a task array and store multiple tasks as an object in that array. see the example code below for how to achieve this
```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

Reply:
I'm storing the tasks in an array but is a property in the class ToDo, where I'm saving my objects. I'm trying to use the class ToDo to make all my procedures.